packages/core/test/financial-templates/PricelessPositionManager.js

```diff
@@ -2,6 +2,7 @@
 const { PositionStatesEnum, didContractThrow } = require("@umaprotocol/common");
 const truffleAssert = require("truffle-assertions");
 const { interfaceName } = require("@umaprotocol/common");
+const { assert } = require("chai");
```

Comment:
nit: do you need to this `assert` import? I thought it came with Truffle for free

Reply:
Nope, my editor just got overzealous :). Nice catch!